### DISCUSSION TITLE
乡巴佬进京

### DISCUSSION THREAD

**Discussion by giscus** [ZexWoo/Giscus#20] [Announcements]
# 乡巴佬进京

终于可以去首都瞧一瞧了。

https://zexwoo.blog/posts/gossip/trip-in-beijing/

**Comment by JourneyHua** [comment]
蚌埠住了 三潭印月不是西湖的么

**Reply by ZexWoo** [reply]
一不小心写反了

**Comment by Cysime** [comment]
拍得很不错啊哈哈哈哈 居然还去了芳草地是没想到的
话说Z老师有没有注意到芳草地旋转门那头猪（

**Reply by ZexWoo** [reply]
哇，没注意到，居然有猪猪？！
